src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -312,6 +312,37 @@
     (future-cancel command-loader))
   context)
 
+;; A map of required postgres settings and their expected value.
+;; PuppetDB will refuse to start unless ALL of these values
+;; match the actual database configuration settings.
+(def required-pg-settings
+  {:standard_conforming_strings "on"})
+
+(defn verify-database-setting
+  "Given m, a map of database settings, verify 'expected', which has the form
+  [setting-name expected-value]."
+  [m expected]
```

Comment:
Could just do `[m [setting value :as expected]]` here, or `[[setting value] expected ...` in the let.  Though in the former case, maybe we don't really need `expected` at all, unless we want it just for clarification.